src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -124,38 +130,44 @@ const addSubtitlesForChecklist = (
   TABLE_MAPPING: TableMappingProps[],
   tasksDone: number
 ): TableMappingProps[] => {
-  const newTableMapping = [...TABLE_MAPPING]
-  newTableMapping[SITE_LAUNCH_TASKS.SET_DNS_TTL - 1].subTitle = (
-    <>
-      {tasksDone === SITE_LAUNCH_TASKS.SET_DNS_TTL - 1 && (
-        <Text fontSize="small">
-          You can check you current DNS TTL through 3rd party applications such
-          as <Link href="https://www.nslookup.io/">nslookup.io</Link>
-        </Text>
-      )}
-    </>
-  )
-  newTableMapping[SITE_LAUNCH_TASKS.DROP_CLOUDFRONT - 1].subTitle = (
-    <>
-      {tasksDone === SITE_LAUNCH_TASKS.DROP_CLOUDFRONT - 1 && (
-        <Text fontSize="small">
-          If you are using CWP, please contact them to do this for you
-        </Text>
-      )}
-    </>
-  )
-  newTableMapping[
-    SITE_LAUNCH_TASKS.DELETE_EXISTING_DNS_RECORDS - 1
-  ].subTitle = (
-    <>
-      {tasksDone === SITE_LAUNCH_TASKS.DELETE_EXISTING_DNS_RECORDS - 1 && (
-        <Text fontSize="small">
-          This should be done as soon as domains are dropped
-        </Text>
-      )}
-    </>
-  )
-  return newTableMapping
+  return TABLE_MAPPING.map((task) => {
+    const newTask = { ...task }
+    if (task.stepNumber === SITE_LAUNCH_TASKS.SET_DNS_TTL - 1) {
+      newTask.subTitle = (
+        <>
+          {tasksDone === SITE_LAUNCH_TASKS.SET_DNS_TTL - 1 && (
+            <Text fontSize="small">
+              You can check you current DNS TTL through 3rd party applications
```

Comment:
```suggestion
              You can check your current DNS TTL through 3rd party applications
```

Reply:
thanks for the catch c9b0362f84fbcb41e2dd885d53437a313b2d6bf1